pom.xml

```diff
@@ -98,9 +98,33 @@
               <additionalClasspathElement>${maven.jdbc.driver.jar}</additionalClasspathElement>
             </additionalClasspathElements>
           </configuration>
-        </plugin> 
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
```

Comment:
That's a step number two in my opinion - I am not sure if this particular setting, which seems to be completely fine for jBPM, will be equally suitable for all other projects. Once we go over more repositories in KIE Group, we will have better overview and for sure this piece of configuration will end up in kie-parent.